### PR TITLE
Fix typos in lib/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: lib/libspl/include/sys/param.h [hunk 1]
```diff
@@ -37,7 +37,7 @@
  * with smaller units (fragments) only in the last direct block.
  * MAXBSIZE primarily determines the size of buffers in the buffer
  * pool. It may be made larger without any effect on existing
- * file systems; however making it smaller make make some file
+ * file systems; however making it smaller make some file
```
[the remark this answers is not in the document]
Comment (Gelma) [reply lib/libspl/include/sys/param.h]: Thanks! Done and squashed

**Comment by Gelma** [comment]
On Thu, Aug 29, 2019 at 02:06:37PM -0700, Ryan Moeller wrote:
> freqlabs requested changes on this pull request.
> 
> 
> 
> > @@ -2081,7 +2081,7 @@ zpool_find_import_impl(libpc_handle_t *hdl, importargs_t *iarg)
>  
>  	/*
>  	 * Process the cache filtering out any entries which are not
> -	 * for the specificed pool then adding matching label configs.
> +	 * for the specified pool then adding matching label configs.
> 
> A few more changes to help clarify this comment:
> ```
> 	 * Process the cache, filtering out any entries which are not
> 	 * for the specified pool. Then, add matching label configs.


If you want I can modify this and push new version.

**Comment by freqlabs** [comment]
> If you want I can modify this and push new version.
After reading it a few more times I changed my mind. Other than possibly adding a comma, I wouldn't change it.

**Comment by behlendorf** [comment]
This LGTM.  @Gelma just let me know if you plan to further tweak this, I'll hold off integrating this change until you let me know it's ready.

**Comment by Gelma** [comment]
> This LGTM. @Gelma just let me know if you plan to further tweak this, I'll hold off integrating this change until you let me know it's ready.

Well, added the comma. Tell me if you like it in this way.

**Pull request merged**